x-pack/plugins/monitoring/server/kibana_monitoring/collectors/get_settings_collector.ts

```diff
@@ -38,11 +40,19 @@ export async function checkForEmailValue(
   }
 }
 
+interface EmailSettingData {
+  xpack: { default_admin_email: string | null };
+}
+
+export interface KibanaSettingsCollector extends Collector<EmailSettingData | undefined> {
+  getEmailValueStructure(email: string | null): EmailSettingData;
+}
+
```

Comment:
Be careful when backporting this, there is a difference between `master` and `7.x` in the `getEmailValueStructure` implementation (the PR does not impact this so it should be fine, but just saying)